CHANGELOG.md

```diff
@@ -63,7 +63,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed `common.util.is_master` function.
 
 ### Fixed
-
+- Fix distributed metrics error.
```

Comment:
```suggestion

- Fix CUDA/CPU device mismatch bug during distributed training for categorical accuracy metric.
```

Reply:
After re-check other metrics, seem the operation is missed only in `CategoricalAccuracy `.
But this operation is done by different method in different classes.
Do you think we need a universal method for all tensors ?